services/cli/src/utils/backtest_rebalance.rs

```diff
@@ -30,83 +36,484 @@ pub async fn rebalancing_simulation(
     simulation_end_epoch: u16,
     steward_cycle_rate: u16,
     number_of_validator_delegations: usize,
+    instant_unstake_cap_bps: u32,
+    validator_historical_start_epoch: u16,
```

Comment:
nit: I would rename this _validator_historical_start_offset_ or something else. The use of the _epoch suffix makes me think this value is an epoch itself, but it's not.